src/basic-map.tsx

```diff
@@ -24,9 +24,14 @@ let DefaultIcon = L.icon({
 L.Marker.prototype.options.icon = DefaultIcon
 /////////////////////end of workaround
 
-const state: Viewport = {
+interface CustomViewport extends Viewport {
+  isAnimatedMarker?: boolean
```

Comment:
Better to add this state into `BasicMap` component. This `state` variable is used only for initial position.

Reply:
but I need isAnimatedMarker only in State interface and its default value(true), if enough for me, because by onzoomstart & onzoomend events in Map I change the value in on the fly.
I need to define this variable only one time in Map's State.